--- /usr/lib/python2.7/site-packages/mwlib/nserve.py
+++ /usr/lib/python2.7/site-packages/mwlib/nserve.py
@@ -276,9 +276,6 @@
         return collection_id
 
     def is_good_baseurl(self, url):
-        netloc = urlparse.urlparse(url)[1].split(':')[0].lower()
-        if netloc == "localhost" or netloc.startswith("127.0.") or netloc.startswith("192.168."):
-            return False
         return True
 
     def _get_params(self, post_data, collection_id):
