diff --git a/programs/server/app/app.js b/programs/server/app/app.js
old mode 100644
new mode 100755
index 77496a26..1119089a
--- a/programs/server/app/app.js
+++ b/programs/server/app/app.js
@@ -77624,6 +77624,33 @@ Meteor.startup(function () {
   }, {
     _id: 'call-management',
     roles: ['admin', 'owner', 'moderator']
+  }, {
+    _id: 'hubot-viva-send-leave-request',
+    roles: ['']
+  }, {
+    _id: 'hubot-viva-approve-leave-request',
+    roles: ['']
+  }, {
+    _id: 'hubot-viva-reject-leave-request',
+    roles: ['']
+  }, {
+    _id: 'hubot-viva-cancel-leave-request',
+    roles: ['']
+  }, {
+    _id: 'hubot-viva-get-leave-requests-list',
+    roles: ['']
+  }, {
+    _id: 'hubot-viva-initiate-new-leave-request-on-behalf-of-user',
+    roles: ['']
+  }, {
+    _id: 'hubot-viva-initiate-new-time-off-request-on-behalf-of-user',
+    roles: ['']
+  }, {
+    _id: 'hubot-viva-work-from-home',
+    roles: ['']
+  }, {
+    _id: 'hubot-viva-set-unset-status-of-being-ill',
+    roles: ['']
   }];
 
   for (const permission of permissions) {
