diff --git a/lib/private/Config.php b/lib/private/Config.php
index 2abf00a..5fa68bf 100644
--- a/lib/private/Config.php
+++ b/lib/private/Config.php
@@ -44,7 +44,23 @@ class Config {
 	const ENV_PREFIX = 'NC_';
 
 	/** @var array Associative array ($key => $value) */
-	protected $cache = array();
+	protected $cache = array(
+		'apps_paths' =>
+		array (
+			0 =>
+			array (
+				'path' => '/var/www/nc/apps',
+				'url' => '/apps',
+				'writable' => false,
+			),
+			1 =>
+			array (
+				'path' => '/var/www/nc/apps2',
+				'url' => '/apps2',
+				'writable' => true,
+			)
+		)
+	);
 	/** @var string */
 	protected $configDir;
 	/** @var string */
