Index: server/lib/private/Config.php
===================================================================
--- server.orig/lib/private/Config.php
+++ server/lib/private/Config.php
@@ -40,7 +40,23 @@ namespace OC;
  */
 class Config {
 	/** @var array Associative array ($key => $value) */
-	protected $cache = array();
+	protected $cache = array(
+		'apps_paths' =>
+		array (
+			0 =>
+			array (
+				'path' => '/var/www/nc/apps',
+				'url' => '/apps',
+				'writable' => false,
+			),
+			1 =>
+			array (
+				'path' => '/var/www/nc/apps2',
+				'url' => '/apps2',
+				'writable' => true,
+			)
+		)
+	);
 	/** @var string */
 	protected $configDir;
 	/** @var string */
