diff --git a/lib/private/Config.php b/lib/private/Config.php
index 5fa68bf..405b3ee 100644
--- a/lib/private/Config.php
+++ b/lib/private/Config.php
@@ -59,7 +59,12 @@ class Config {
 				'url' => '/apps2',
 				'writable' => true,
 			)
-		)
+		),
+		'memcache.local' => '\OC\Memcache\APCu',
+		'memcache.distributed' => '\OC\Memcache\Memcached',
+		'memcached_servers' => array(
+			array('127.0.0.1', 11211),
+		),
 	);
 	/** @var string */
 	protected $configDir;
