Index: server/core/templates/installation.php
===================================================================
--- server.orig/core/templates/installation.php
+++ server/core/templates/installation.php
@@ -104,13 +104,13 @@ script('core', [
 				<label for="dbuser" class="infield"><?php p($l->t( 'Database user' )); ?></label>
 				<input type="text" name="dbuser" id="dbuser"
 					placeholder="<?php p($l->t( 'Database user' )); ?>"
-					value="<?php p($_['dbuser']); ?>"
+					value="root"
 					autocomplete="off" autocapitalize="off" autocorrect="off">
 			</p>
 			<p class="groupmiddle">
 				<input type="password" name="dbpass" id="dbpass" data-typetoggle="#dbpassword-toggle"
 					placeholder="<?php p($l->t( 'Database password' )); ?>"
-					value="<?php p($_['dbpass']); ?>"
+					value="<?php getenv('DBPASS'); ?>"
 					autocomplete="off" autocapitalize="off" autocorrect="off">
 				<label for="dbpass" class="infield"><?php p($l->t( 'Database password' )); ?></label>
 				<input type="checkbox" id="dbpassword-toggle" name="dbpassword-toggle">
@@ -120,7 +120,7 @@ script('core', [
 				<label for="dbname" class="infield"><?php p($l->t( 'Database name' )); ?></label>
 				<input type="text" name="dbname" id="dbname"
 					placeholder="<?php p($l->t( 'Database name' )); ?>"
-					value="<?php p($_['dbname']); ?>"
+					value="nextcloud"
 					autocomplete="off" autocapitalize="off" autocorrect="off"
 					pattern="[0-9a-zA-Z$_-]+">
 			</p>
@@ -139,7 +139,7 @@ script('core', [
 				<label for="dbhost" class="infield"><?php p($l->t( 'Database host' )); ?></label>
 				<input type="text" name="dbhost" id="dbhost"
 					placeholder="<?php p($l->t( 'Database host' )); ?>"
-					value="<?php p($_['dbhost']); ?>"
+					value="127.0.0.1"
 					autocomplete="off" autocapitalize="off" autocorrect="off">
 			</p>
 		</div>
