diff --git a/core/Controller/SetupController.php b/core/Controller/SetupController.php
index ab8b197..e878cb6 100644
--- a/core/Controller/SetupController.php
+++ b/core/Controller/SetupController.php
@@ -113,7 +113,7 @@ class SetupController {
 			}
 		}
 
-		header('Location: ' . \OC::$server->getURLGenerator()->getAbsoluteURL('index.php/core/apps/recommended'));
+		header('Location: ' . \OC::$server->getURLGenerator()->getAbsoluteURL('index.php/core/apps/defaultPageUrl'));
 		exit();
 	}
 
diff --git a/core/src/jquery/showpassword.js b/core/src/jquery/showpassword.js
index a00a57c..5b29532 100644
--- a/core/src/jquery/showpassword.js
+++ b/core/src/jquery/showpassword.js
@@ -97,6 +97,7 @@ $.fn.extend({
 
 			// Create clone
 			const $clone = cloneElement($input)
+			$clone.val($input.val())
 			$clone.insertAfter($input)
 
 			// Set callback arguments
diff --git a/core/templates/installation.php b/core/templates/installation.php
index 2affefe..66d1ef9 100644
--- a/core/templates/installation.php
+++ b/core/templates/installation.php
@@ -58,7 +58,7 @@ script('core', 'install');
 	<?php endif; ?>
 
 	<?php if (!$_['directoryIsSet'] or count($_['errors']) > 0): ?>
-	<fieldset id="datadirField">
+	<fieldset id="datadirField" style="display: none">
 		<div id="datadirContent">
 			<label for="directory"><?php p($l->t('Data folder')); ?></label>
 			<input type="text" name="directory" id="directory"
@@ -70,7 +70,7 @@ script('core', 'install');
 	<?php endif; ?>
 
 	<?php if (!$_['dbIsSet'] or count($_['errors']) > 0): ?>
-	<fieldset id='databaseBackend'>
+	<fieldset id='databaseBackend' style="display: none">
 		<?php if ($_['hasMySQL'] or $_['hasPostgreSQL'] or $_['hasOracle']) {
 			$hasOtherDB = true;
 		} else {
@@ -97,18 +97,18 @@ script('core', 'install');
 	</fieldset>
 
 		<?php if ($hasOtherDB): ?>
-		<fieldset id='databaseField'>
+		<fieldset id='databaseField' style="display: none">
 		<div id="use_other_db">
 			<p class="grouptop">
 				<label for="dbuser"><?php p($l->t('Database user')); ?></label>
 				<input type="text" name="dbuser" id="dbuser"
-					value="<?php p($_['dbuser']); ?>"
+					value="root"
 					autocomplete="off" autocapitalize="none" autocorrect="off">
 			</p>
 			<p class="groupmiddle">
 				<label for="dbpass"><?php p($l->t('Database password')); ?></label>
 				<input type="password" name="dbpass" id="dbpass"
-					value="<?php p($_['dbpass']); ?>"
+					value="secret"
 					autocomplete="off" autocapitalize="none" autocorrect="off">
 				<button id="show" class="toggle-password" aria-label="<?php p($l->t('Show password')); ?>">
 					<img src="<?php print_unescaped(image_path('', 'actions/toggle.svg')); ?>" alt="<?php p($l->t('Toggle password visibility')); ?>">
@@ -117,7 +117,7 @@ script('core', 'install');
 			<p class="groupmiddle">
 				<label for="dbname"><?php p($l->t('Database name')); ?></label>
 				<input type="text" name="dbname" id="dbname"
-					value="<?php p($_['dbname']); ?>"
+					value="nc"
 					autocomplete="off" autocapitalize="none" autocorrect="off"
 					pattern="[0-9a-zA-Z$_-]+">
 			</p>
@@ -134,7 +134,7 @@ script('core', 'install');
 			<p class="groupbottom">
 				<label for="dbhost"><?php p($l->t('Database host')); ?></label>
 				<input type="text" name="dbhost" id="dbhost"
-					value="<?php p($_['dbhost']); ?>"
+					value="mariadb:3306"
 					autocomplete="off" autocapitalize="none" autocorrect="off">
 			</p>
 			<p class="info">
